From 38d7e37828076c4712f0b0f9c7c96c849c8e3aa8 Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Wed, 1 Jun 2016 17:35:25 +0800
Subject: [PATCH 1080/5242] MLK-12864: ARM: dts: imx6ull-14x14-ddr3-arm2:
 enable WDOG_B reset

commit  4278926f5205440ce943292ac0ed24e4f6fd7eec from
https://source.codeaurora.org/external/imx/linux-imx.git

enable WDOG_B reset pin on i.mx6ull-14x14-ddr3-arm2 board

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |   11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index fcd1c84..67e4e87 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -841,6 +841,11 @@
 			>;
 		};
 
+		pinctrl_wdog: wdoggrp {
+			fsl,pins = <
+				MX6UL_PAD_GPIO1_IO08__WDOG1_WDOG_B	0x30b0
+			>;
+		};
 	};
 };
 
@@ -1005,3 +1010,9 @@
 	vmmc-supply = <&reg_sd2_vmmc>;
 	status = "okay";
 };
+
+&wdog1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_wdog>;
+	fsl,ext-reset-output;
+};
-- 
1.7.9.5

